src/sscg.c

```diff
@@ -529,15 +570,6 @@ main (int argc, const char **argv)
         }
     }
 
```

Comment:
Is it intentional that there are no more checks on the key size?
Perhaps it's an idea to make sure that it's actually bigger or equal to the required level, in case someone manually sets key size to 1024?

Reply:
Hmm, fair point. I suppose I probably should enforce the minimum.

I'll note the minimum in the `--help` content too.

Reply:
Maybe also add an option to override the minimum check?  Always good to give folks rope...

Reply:
@jamescassell I think I'd prefer not to add that option. I'm reasonably comfortable that anyone who has a strong-enough need to create an insecure certificate can find other tools to do so.